lib/content-services/content-node-share/content-node-share.dialog.ts

```diff
@@ -68,21 +80,26 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
         private nodesApiService: NodesApiService,
         private contentService: ContentService,
         private renditionService: RenditionsService,
-        @Inject(MAT_DIALOG_DATA) public data: any) {
-    }
+        private notificationService: NotificationService,
```

Comment:
didn't we decide to not use anymore the NotificationService is the component but wires it in the apps?

Reply:
Ok, will move it to demo shell.